packages/console/src/command-history/CommandHistoryItemTooltip.scss

```diff
@@ -1,23 +1,36 @@
 @import '../../../components/scss/custom.scss';
 .command-history-item-popper {
   padding: 0;
+  .popper-content .tooltip-content {
+    padding: 0;
+  }
   .command-history-item-tooltip {
+    display: flex;
+    flex-direction: column;
     max-height: 50vh;
     max-width: 50vw;
-    overflow: auto;
     text-align: left;
-    padding: $tooltip-padding-y $tooltip-padding-x;
-    .result-info {
-      text-align: right;
-    }
-    .time-string {
-      color: $text-muted;
+    .scroll-container {
+      padding: $spacer-2;
+      overflow: auto;
+      flex: 0 1 auto;
     }
-    hr {
-      background-color: $gray-400;
-    }
-    .error-message {
-      color: $danger;
+    .result-info {
+      flex: 1 0 auto;
```

Comment:
Why should this grow, but not the content? Shouldn't the popper have no empty space to fill?

Reply:
Test it visually and see if any combo looks weird. It should look good for all cases should be good if long messaga + short error (DHC), long message + long error (DHE), long message + no error, short message + long error, short message + short error (DHC)

.scroll-container will still claim space until .result + the parent height hits the 50vh max-height, then it will get scroll. There should still be no empty space to fill.

flex-grow only controls the case when the parent max-height is reached, at that point, however we don't want them do grow proportionally, we want the .result to grow to claim as much space as required for the .error to a max of 25vh.

Ideally .result should be able to grow the full height if scroll-container is only 1 line, but I couldn't figure out a way.

Reply:
flex-grow should only have an affect if the parent is bigger than the content of the children, no? I changed it to `0 0 auto` and didn't notice any differences

Doesn't really affect anything, I was just curious

Reply:
You are correct, technically I just need the max-height on the result, and the flex-shrink, the rest can be left as default.